src/pybind/mgr/rook/module.py

```diff
@@ -82,6 +88,18 @@ class RookOrchestrator(MgrModule, orchestrator.Orchestrator):
             default='local',
             desc='storage class name for LSO-discovered PVs',
         ),
+        Option(
+            'prometheus_tls_secret_name',
+            type='str',
+            default='rook-ceph-prometheus-server-tls',
+            desc='name of tls secret in k8s for prometheus',
+        ),
+        Option(
+            'dashboard_tls_secret_name',
+            type='str',
+            default='rook-ceph-dashboard-server-tls',
+            desc='name of tls secret in k8s for dashboard',
+        ),
```

Comment:
the dashboard already has a service that can be enabled with `https`, what's the point of providing this secret here? are you planning to secure internal communications as well?

Reply:
yes, idea is to have secure internal communication

Reply:
We can remove this if unnecessary.

Reply:
removed it